concepts/bit-manipulation/links.json

```diff
@@ -0,0 +1,14 @@
+[
+  {
+    "url": "https://crystal-lang.org/reference/1.10/syntax_and_semantics/operators.html#binary",
```

Comment:
I thinked you missed making this not version specfic docs, e.g "https://crystal-lang.org/reference/syntax_and_semantics/operators.html#binary"